jekyll/_cci2_ja/collect-test-data.md

```diff
@@ -14,7 +14,7 @@ CircleCI は、XML ファイルからテストメタデータを収集し、そ
 
 テスト結果をアーティファクトとして表示するには、`store_artifacts` ステップを使用してテスト結果をアップロードします。
 
-コンフィグでは、以下のように `[store_test_results]({{site.baseurl}}/ja/2.0/configuration-reference/#store_test_results)` キーが使用されます。
+コンフィグでは、以下のように [store_test_results]({{site.baseurl}}/ja/2.0/configuration-reference/#store_test_results) キーが使用されます。
```

Comment:
```suggestion
コンフィグでは、以下のように [`store_test_results`]({{site.baseurl}}/ja/2.0/configuration-reference/#store_test_results) キーが使用されます。
```

Thanks for catching this! We just need to have `store_test_results` itself wrapped with a backtick. I suggested this as a suggestion, which you can accept in the PR, or just push the change in another commit!

Thank you for contributing to our docs!